src/modules/common/transaction.options.ts

```diff
@@ -0,0 +1,15 @@
+import { TokenTransfer } from '@multiversx/sdk-core/out';
+
+export class TransactionOptions {
```

Comment:
I think that a better naming would be `TransactionIntent`

Reply:
I would rename it in a separate PR once all the other PRs upstream get merged